CONTRIBUTING.md

```diff
@@ -357,6 +357,14 @@ See the [Fixing an Issue](#fixing-an-issue) section above for further details.
 Be sure to include unit and integration tests for your new resource under the Tests folder as well as an example under the examples folder.
 Please also add a full description of your new resource and its parameters to the module's README.
 
+### Resource Naming
```

Comment:
Should we also add how we name examples, tests. If they should also start with `MSFT_`, like it's resource, or if they don't need to start with `MSFT_'. I might not be necessary for them to be named with `MSFT_`, like the resource need to be. But for clarity when referencing files, it might be good to have the same name on everything? Or on the other hand, might be good _not_ to have the `MSFT_`on other files?

And helper files and "localization-string" files how should they be named? Probably prefixed with its module name. But does it say that somewhere? If not, add it here to maybe? 

_This comment is based of an comment raised by @lugilink in PowerShell/xSqlServer#137. I took liberty to 'forward' it here when I saw this PR._